multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -844,7 +848,10 @@ def fit(
         self._fit(**_fit_args)
         training_end = time.time()
         self._total_train_time = training_end - training_start
-        logger.info(f"Models and intermediate outputs are saved to {self._save_path} ")
+
+        # TODO(?) We should have a separate "_post_training_event()" for logging messages.
+        logger.info(get_fit_complete_message(self._save_path))
```

Comment:
Is `info` shown by default or only if the user increases verbosity? In what situation will the user see the new logs added in this PR?

Reply:
It's shown by default (verbosity=2,3).

Reply:
@Innixma Our logger seems to have a bug. Setting verbosity=4 doesn't print the messages in `logger.debug()`.

Reply:
@zhiqiangdon Can you create a GitHub issue with a minimal reproducible example with the logger in isolation?